content/docs/localization/joining-the-localization-team.md

```diff
@@ -1,46 +1,115 @@
-- It may take a few hours/days before your request is accepted.
-- If you get accepted, you may add yourself (or be added) to [the website's credits page](/credits).
-- Legal stuff: your translations will be licensed under the same license as Scratch Addons (GPL v3.0)
+We support most languages that Scratch itself supports.
+
+<details>
+<summary>Click to access the full list supported languages (50+ languages)</summary>
+Abkhaz (ab), Afrikaans (af), Amharic (am), Aragonese (an), Arabic (ar), Azerbaijani (az), Belarusian (be), Bulgarian (bg), Bengali (bn), Catalan (ca), Central Kurdish (ckb), Czech (cs), Welsh (cy), Danish (da), German (de), Greek (el), Spanish (es), Estonian (et), Basque (eu), Persian (fa), Finnish (fi), French (fr), Western Frisian (fy), Irish (ga), Gaelic, Scottish (gd), Galician (gl), Hebrew (he), Croatian (hr), Haitian (Haitian Creole) (ht), Hungarian (hu), Armenian (hy), Indonesian (id), Icelandic (is), Italian (it), Japanese (ja), Georgian (ka), Kazakh (kk), Khmer (km), Korean (ko), Kurdish (ku), Lithuanian (lt), Latvian (lv), Maori (mi), Mongolian (mn), Norwegian Bokmål (nb), Dutch (nl), Norwegian Nynorsk (nn), Northern Sotho (nso), Odia (or), Polish (pl), Portuguese (Brazil) (pt_BR), Quechua (qu), Rapa Nui (rap), Romanian (ro), Russian (ru), Slovak (sk), Slovenian (sl), Serbian (sr), Swedish (sv), Swahili (sw), Thai (th), Tswana (tn), Turkish (tr), Ukrainian (uk), Uzbek (uz), Vietnamese (vi), Xhosa (xh), Chinese (China) (zh_CN), Chinese (Taiwan) (zh_TW), Zulu (zu).
+</details>
+
+We do not support these languages yet: Asturianu (ast), Esperanto (eo), Español Latinoamericano (es_419), Filipino (fil), Hausa (ha), Japanese (Hiragana) (ja-Hira), Occitan (oc), Portuguese (Portugal) (pt_PT). If you're interested on becoming a translator for one of these languages we currently do not support, please send an email to `translation@scratchaddons.com`.
+
+These languages **do not** accept new translator requests: **Italian, Japanese, Portuguese, Slovenian, Spanish.**
+
+## Steps to join as a translator
+
+Thanks for wanting to help with translating Scratch Addons to other languages! Please follow the steps below.
+
+#### 1. Verify your Scratch account
+1. Visit this [Scratch studio](https://scratch.mit.edu/studios/33665222/comments) with your Scratch account.
+1. Send a studio comment indicating the language you want to translate to (for example, `Greek`).
+![Screenshot of Scratch](/assets/img/docs/scratch-req-language.png)
+
+If you don't own a Scratch account, please send an email to `translation@scratchaddons.com` instead.
 
-### Any questions?
+#### 2. Create a Transifex account
+1. Go to [Transifex Sign Up](https://app.transifex.com/signup/).  
+1. You will be shown Transifex's "Sign Up" screen. **DO NOT sign up with Google.**  
+1. In the "username" field, type in `scratch_` **followed by your Scratch username.**  
+For example, if your Scratch username is `john_walker954`, type `scratch_john_walker954`.

```

Comment:
This kind of username is... not nice. If I were you, I would suggest firstly to "put your Scratch username as your username", then suggest this if you really need it.

Oh, and as a **suggestion**, not a necessity.

Reply:
> This kind of username is... not nice.

Justify your answer.
But yeah, this could be changed to an explicit recommendation, instead of being written like a requirement.

Reply:
You need a justification? Why should I name myself "John Doe from Scratch" when can I just name myself "John Doe", provided that "John Doe" is my username?

Reply:
It could simplify rejecting people that didn't fully read the instructions. But yeah, it's not worth it to have this as a strict requirement, as we want to allow people with existing Transifex accounts to join too.

I'll change this

Reply:
https://github.com/ScratchAddons/website-v2/pull/363/commits/caa3bb0498164cfa251644d09d9a31be9e0b928e